monitor.py

```diff
@@ -18,13 +18,13 @@
 if TYPE_CHECKING:
     import tkinter
 
-import config as conf_module  # Necessary to see the same config.trace_on as elsewhere
 import util_ships
 from config import config
 from edmc_data import edmc_suit_shortnames, edmc_suit_symbol_localised
 from EDMCLogging import get_main_logger
 
 logger = get_main_logger()
+STARTUP = 'startup'
```

Comment:
Now I'm having thoughts about all such constants being defined in `EDMCLogging`.

And if they're part of a dict, set, list, whatever we *might* even be able to list them all on `--help`.

They should certainly have a mandated prefix on the name if not part of some data structure.

Reply:
do we want to couple stuff that much? I didnt really want to make it super constrained, they were just there as a "I dont want to type 'startup' 30 times"

Reply:
Saving a whole two characters every time 🤣 OK.